index.html

```diff
@@ -926,26 +920,26 @@
     </p>
     <ul>
       <li>
-        Only <a>browsing context</a>s with a defined <a>origin</a>, whose schema
-        is starting with <code>"https"</code> must be able to access
+        Only <code>
+        <a href="https://html.spec.whatwg.org/#document">Document</a></code>s
```

Comment:
Yes, we should be pedantic. We need a Window because the visibility state and focus, need a Document for origin, and a script execution environment. This ought to be possible to express in a more concise way, but anyway you are right.